app/gateways/http/authentication/create_token.go

```diff
@@ -0,0 +1,90 @@
+package authentication
+
+import (
+	"encoding/json"
+	auth "github.com/patriciapedrosaa/transfer-me/app/domain/authentication"
+	http_server "github.com/patriciapedrosaa/transfer-me/app/gateways/http"
+	"net/http"
+)
+
+var (
+	ErrInvalidPayload = "invalid request payload"
+	ErrRequiredFields = "invalid fields"
+	ErrNotFound       = "not found"
+	ErrUnexpected     = "something went wrong"
+)
+
+type LoginRequest struct {
+	CPF    string `json:"cpf" validate:"required"`
+	Secret string `json:"secret" validate:"required"`
+}
+
+type CreateTokenResponse struct {
+	Token string `json:"Token" validate:"required"`
+}
+
+type LoginResponse struct {
+	Token string
+}
+
+func (h Handler) CreateToken(w http.ResponseWriter, r *http.Request) {
+	var body LoginRequest
+	decoder := json.NewDecoder(r.Body)
+	decoder.DisallowUnknownFields()
+
+	if err := decoder.Decode(&body); err != nil {
+		responseError(w, http.StatusBadRequest, ErrInvalidPayload)
+		return
+	}
+
+	validator := http_server.NewJSONValidator()
+	err := validator.Validate(body)
+	if err != nil {
+		responseError(w, http.StatusBadRequest, ErrRequiredFields)
+		return
+	}
+
+	defer r.Body.Close()
+
+	account, err := h.accountUseCase.GetByCpf(body.CPF)
+	if err != nil {
+		responseError(w, http.StatusNotFound, ErrNotFound)
+		return
+	}
```

Comment:
nice catch, but I guess `Not Found` is not a good status neither as HTTP status nor err message, since this way you'll be enumerating your account's CPF. Bad request as HTTP Status and a non-enumerating message will be better